website/docs/d/schema_template_bd.html.markdown

```diff
@@ -40,7 +40,15 @@ data "mso_schema_template_bd" "bridge_domain" {
 * `optimize_wan_bandwidth` - (Optional) Boolean flag to enable or disable the wan bandwidth optimization.
 * `layer2_stretch` - (Optional) Boolean flag to enable or disable the layer-2 stretch.
 * `layer3_multicast` - (Optional) Boolean flag to enable or disable layer 3 multicast traffic.
-* `dhcp_policy` - (Optional) Map to provide dhcp_policy configurations. 
+* `dhcp_policies` - (Optional) Block to provide dhcp_policy configurations. 
+* `dhcp_policies.name` - (Optional) Dhcp_policy name. Required if you specify the dhcp_policy.
+* `dhcp_policies.version` - (Optional) Version of dhcp_policy. Required if you specify the dhcp_policy.
+* `dhcp_policies.dhcp_option_policy_name` - (Optional) Name of dhcp_option_policy. 
+* `dhcp_policies.dhcp_option_policy_version` - (Optional) Version of dhcp_option_policy.
```

Comment:
Please change the documentation format.
For your reference: https://github.com/CiscoDevNet/terraform-provider-aci/pull/857/files#diff-0505816fc7d3615c5e4a081b8c2cf3e731a4b18fdb4d76f12bcf6e67ac7d3de1R107-R111

Reply:
ok